src/main/java/core/basesyntax/service/impl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportGenerator;
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    private static final String FRUIT = "fruit";
+    private static final String QUANTITY = "quantity";
+    private static final String SEPARATOR = ",";
+
+    @Override
+    public String generateReport() {
+        StringBuilder report = new StringBuilder();
+        report.append(FRUIT).append(SEPARATOR).append(QUANTITY).append(System.lineSeparator());
+        for (Map.Entry<String, Integer> entry : Storage.storage.entrySet()) {
+            report.append(entry.getKey()).append(SEPARATOR).append(entry.getValue())
+                    .append(System.lineSeparator());
+        }
+        return report.toString();
+    }

```

Comment:
According to the checklist, you should not begin the class or method implementation with an empty line. Please remove the redundant empty lines at the beginning of the class and method.